superset-frontend/src/explore/components/RowCountLabel.tsx

```diff
@@ -31,11 +31,20 @@ const propTypes = {
   loading: PropTypes.bool,
 };
 
+type RowCountLabelProps = {
+  rowcount?: number;
+  limit?: number;
+  rows?: string;
```

Comment:
I don't think we use prop `rows` here 🤔

Reply:
You're right. I tried not to change more widely (this field was previously in propTypes) to facilitate the review, but - taking into account your comments - I will introduce such optimizations in future migrations as well.

Thanks for your comments. They are very valuable to me. At the beginning, I just wanted to touch small things to calibrate myself to the style, the expected unit of work in the project, so that our cooperation would go smoothly. There are a lot of components to migrate and I don't want to get stuck in a big PR review.